src/components/OLDTodoList.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div>
+   <v-btn @click="onShowForm" color="blue">Add Todo</v-btn>
+   <v-data-table :headers="headers" :items="todos" class="elevation-1">
+      <template v-slot:body="{ items }">
+        <tr v-for="(item, index) in items" :key="index">
+          <td>{{ item.task }}</td>
+          <td>{{ item.description }}</td>
+          <td>{{ item.assignedTo }}</td>
+          <td>
+            <v-btn color="blue" small @click="editTask(item, index)">
+              Edit
+            </v-btn>
+          </td>
+          <td>
+            <v-btn color="red" small @click="deleteTask(index)">
+              Delete
+            </v-btn>
+          </td>
+        </tr>
+      </template>
+    </v-data-table>
+    <v-dialog
+        v-model="dialog"
+             persistent
+              max-width="290">
+              <v-card>
+              <v-card-title class="text-h5">
+                DO you want to detele this ?
+              </v-card-title>
+              <v-card-actions>
+              <v-spacer></v-spacer>
+              <v-btn
+              color="green"
+              @click="dialog=false">
+              Cancel
+              </v-btn>
+              <v-btn
+              color="red"
+              @click="deleteConfirmed ()">
+              Delete
+              </v-btn>
+             </v-card-actions>
+             </v-card>
+            </v-dialog>
```

Comment:
Move this to a `DeleteDialog.vue` file and import it to this file; so that it can be called from anywhere.